src/styles/blocks/reset.scss

```
@@ -0,0 +1,4 @@
+.body,
+.nav__list {
+  margin: 0;
+}
```

Set resets to tags no to classes.
```suggestion
.body,
ul {
  margin: 0;
}
```